src/SFA.DAS.RoatpOversight.Web/Controllers/OversightController.cs

```diff
@@ -32,88 +33,63 @@ public async Task<IActionResult> Applications()
         }
 
         [HttpGet("Oversight/Outcome/{applicationId}")]
-        public IActionResult Outcome(Guid applicationId)
+        public async Task<IActionResult> Outcome(Guid applicationId)
         {
-            var stubbedViewModel = GetStubbedViewModel();
-            var applicationDetails = stubbedViewModel.ApplicationDetails.FirstOrDefault();
-            var vm = new OutcomeViewModel
-            {
-                ApplicationId = applicationId,
-                ApplicationReferenceNumber =  applicationDetails.ApplicationReferenceNumber,
-                ApplicationSubmittedDate =  applicationDetails.ApplicationSubmittedDate,
-                OrganisationName =  "THIS IS A DUMMY PAGE",
-                ProviderRoute = applicationDetails.ProviderRoute,
-                Ukprn =  applicationDetails.Ukprn
-            };
-
+            var vm = await _orchestrator.GetOversightDetailsViewModel(applicationId);
 
             return View(vm);
         }
 
         [HttpPost("Oversight/Outcome/{applicationId}")]
-        public IActionResult EvaluateOutcome(Guid applicationId, string status)
+        public async Task<IActionResult> EvaluateOutcome(Guid applicationId, string status)
         {
-            var stubbedViewModel = GetStubbedViewModel();
-            var applicationDetails = stubbedViewModel.ApplicationDetails.FirstOrDefault();
-            var viewModel = new OutcomeViewModel
-            {
-                ApplicationReferenceNumber = applicationDetails.ApplicationReferenceNumber,
-                ApplicationSubmittedDate = applicationDetails.ApplicationSubmittedDate,
-                OrganisationName = "THIS IS A DUMMY PAGE",
-                ProviderRoute = applicationDetails.ProviderRoute,
-                Ukprn = applicationDetails.Ukprn
-            };
+            var errorMessages = OverallOutcomeValidator.ValidateOverallOutcome(status);
+            var viewModel = await _orchestrator.GetOversightDetailsViewModel(applicationId);
 
-            viewModel.ErrorMessages = new List<ValidationErrorDetail>();
-            if (string.IsNullOrEmpty(status))
+            if (errorMessages.Any())
             {
-                viewModel.ErrorMessages.Add(new ValidationErrorDetail
-                    {ErrorMessage = "error message", ValidationStatusCode = new ValidationStatusCode()});
-
+                    viewModel.ErrorMessages = errorMessages;
                 return View($"~/Views/Oversight/Outcome.cshtml", viewModel);
             }
 
             if (status.ToLower() == "successful")
             {
+
                 var viewModelSuccessful = new OutcomeSuccessfulViewModel
```

Comment:
Can these view models be merged? The only difference I can see is the outcome status

Reply:
Yes, will do